cypress/integration/distroSettings/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const clickSave = () => {
+  cy.dataCy("save-settings-button")
+    .should("not.have.attr", "aria-disabled", "true")
+    .click();
+};
```

Comment:
ngl I was being lazy when I wrote this teehee... but project settings has the same function in [`constants.ts`](https://github.com/evergreen-ci/spruce/blob/bd3fe088a3a6e926e0d398879ce05adecbf87986/cypress/integration/projectSettings/constants.ts#L41-L45) so maybe we should share this method